setup.cfg

```
@@ -2,3 +2,14 @@
 max-line-length = 88
 ignore = E203, E266, E501, W503
 select = B,C,E,F,W,T4,B9
+per-file-ignores =
+    versioneer.py:T001
+    unidist/_version.py:T001
```

We can just skip autogenerated files, there is no point to check them:
```suggestion
exclude =
    versioneer.py,
    unidist/_version.py
```

---

I think we can keep as it is. There are no many different warnings. If we want to keep the whole code base formatted, we likely want to check these too.